docs/osr-resources/InnerSource.md

```diff
@@ -17,24 +17,25 @@ Presentation by Chamindra de Silva to FINOS Members Meeting on June 15
         alt="Image of slide number 1" />
     </div>
     <div class="slide-notes">
-      <p>Hello everyone, my name is Chamindra and I am a InnerSource
+      <p>Hello everyone, my name is Chamindra and I am an InnerSource
         Product Owner and Solution Architect at Citi. I am also a
-        co-lead of the FinOS InnerSource Special Interest Group.</p>
-      <p>For those who are not aware InnerSource is the application
+        co-lead of the FINOS InnerSource Special Interest Group.</p>
+      <p>For those who are not aware InnerSource, it is the application
         of Open Source principles and working culture within the
         boundary of an enterprise. Large enterprises in particular
-        suffer from many organizational knowledge silos and Open Source
-        can provide a lot of benefit to improve quality and
-        reduce cost.</p>
-      <p>Before joining Citi I led two open source projects in
-        humanitarian response and education and was a committer to Apache
-        Web Services, so I had some exposure to open source and the Apache
-        Way going into Citi.</p>
+        suffer from many organizational knowledge silos and Open Source culture
+        can provide a lot of benefit to improve engineering quality and
+        reduce cost through reuse, though those product might never venture 
+        outside the boundaries of that institution.</p>
+      <p>Before joining Citi I led two Open Source projects in
+        humanitarian response and education sector and was a committer to Apache
+        Web Services, so I had some exposure to Open Source and the Apache
+        Way going into the Banking Industry.</p>
       <p>But I understood that there are nuances in InnerSource
-        that make it different from open source and it needs to be
+        that make it different from Open Source and it needs to be
```

Comment:
It's more house style but also generally accepted that open source isn't capitalised, so that's what we go with here